src/jvmTest/kotlin/it/krzeminski/githubactionstyping/JsonSchemaValidatorSchemaValidationTest.kt

```diff
@@ -46,7 +48,12 @@ private fun beValid(): Matcher<File> {
         var errorMessage: String? = null
         MatcherResult(
             runCatching {
-                schema.validate(dataFile.inputStream().use { Load().loadOne(it) }.toJsonElement()) {
+                schema.validate(dataFile.inputStream().use {
+                    Load(
+                        // work-around for https://github.com/krzema12/snakeyaml-engine-kmp/pull/390
```

Comment:
```suggestion
                        // work-around for https://github.com/krzema12/snakeyaml-engine-kmp/pull/390
                        // Per https://yaml.org/spec/1.2.2/#recommended-schemas, the Core Schema is
                        // the recommended default schema that YAML processor should use unless
                        // instructed otherwise.
```